functions/go/set-labels/transformer/labels_test.go

```diff
@@ -54,6 +54,11 @@ spec:
     - metadata:
         labels:
           testkey: testvalue
+          app: testvalue
+          env: testvalue
+          quotedBoolean: testvalue
+          quotedFruit: testvalue
+          unquotedBoolean: testvalue
 `
 
 	sameLabelInput := `
```

Comment:
The apiVersion for ConfigMap should be `v1`.
Please audit the file ensure all occurrences have been updated.